packages/react-utilities/package.json

```diff
@@ -29,7 +29,7 @@
     "@fluentui/eslint-plugin": "^1.0.1",
     "@fluentui/jest-serializer-merge-styles": "^8.0.2",
     "@fluentui/scripts": "^1.0.0",
-    "@testing-library/react-hooks": "^5.0.3",
+    "@testing-library/react-hooks": "5.1.0",
```

Comment:
Just wondering: is there a reason to bump this version?

Reply:
I haven't bumped it. must've been some sync script or in the rebasing

Reply:
It's actually the `syncpack:fix` that's bumping the version